src/coreclr/vm/callstubgenerator.cpp

```diff
@@ -1902,4 +1902,122 @@ CallStubGenerator::ReturnType CallStubGenerator::GetReturnType(ArgIterator *pArg
     return ReturnTypeVoid;
 }
 
+static CallStubHeader *UpdateCallStubForMethod(MethodDesc *pMD)
```

Comment:
What was the reason for moving these methods to the callstubgenerator? I would prefer keeping only the implementation oof the CallStubGenerator class in this file.

Reply:
Maybe `nativecodeinvocation.cpp`, similar to `reflectioninvocation.cpp` naming.